src/lib.rs

```diff
@@ -90,9 +45,9 @@ impl<T: Copy + PartialOrd + Debug> CKMS<T> {
     /// ```
     /// use quantiles::CKMS;
     ///
-    /// let mut ckms = CKMS::<u16>::new(0.001);
+    /// let mut ckms = CKMS::<u64>::new(0.001);
```

Comment:
punting on this conversation for now, which was taken offline and irrelevant to this review.